proxy/scripts/test_upgrade.sh

```diff
@@ -1,13 +1,14 @@
 #!/usr/bin/env bash
 
 DEPLOYED_TAG="$(cat "$GITHUB_WORKSPACE"/proxy/DEPLOYED)"
+VERSION_TAG="$(cat "$GITHUB_WORKSPACE"/VERSION)"
 DEPLOYED_VERSION="$(echo "$DEPLOYED_TAG" | cut -d '-' -f 1)"
 DEPLOYED_DIR=$GITHUB_WORKSPACE/deployed-proxy/
 
 
 git clone --branch "$DEPLOYED_TAG" "https://github.com/$GITHUB_REPOSITORY.git" "$DEPLOYED_DIR"
 
-npx ganache-cli --gasLimit 9000000 --quiet --allowUnlimitedContractSize &
```

Comment:
Does it not fit to 9M?

Reply:
Unexpectedly changed - thought it was a gas limit issue - but it is not - can downgrade it